sorc/emcsfc_ice_blend.fd/CMakeLists.txt

```diff
@@ -5,9 +5,9 @@ set(exe_name emcsfc_ice_blend)
 add_executable(${exe_name} ${fortran_src})
 target_link_libraries(
   ${exe_name}
-  bacio_4
-  g2_4
-  w3nco_4
+  bacio::bacio_4
+  g2::g2_4
+  w3nco::w3nco_4
   ${JASPER_LIBRARIES}
   PNG::PNG
   ZLIB::ZLIB)
```

Comment:
ZLIB, Japser and PNG are not direct dependencies, Can be removed.

Reply:
done.